CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+project(Arbiter C CXX)
+
+cmake_minimum_required(VERSION 3.0)
```

Comment:
That's a good idea, although it looks like Travis CI doesn't have a very modern CMake installed by default on Linux machines.